Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu
```

Comment:
can we build this without registry access?

Reply:
That is the official registry for atomic images. Doesn't require any login.